incubator/geth/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: "arilot/geth"
-  tag: "alpine"
+  tag: "latest"
```

Comment:
Use a stable image

Reply:
fixed. also I have change image to official